third-party/CMakeLists.txt

```diff
@@ -68,22 +68,45 @@ message(STATUS "Install bundled dependencies into ${DEPS_INSTALL_DIR}")
 
 set(HYBRIDSQL_ASSERTS_HOME https://github.com/4paradigm/hybridsql-asserts)
 
+function(get_linux_lsb_release_information)
+    find_program(LSB_RELEASE_EXEC lsb_release)
+    if(NOT LSB_RELEASE_EXEC)
```

Comment:
if `lsb_release` is not found, can parse `/etc/os-release` to see the release

Reply:
That file seems just work, I'm gona rm lsb_release